eslint.config.js

```diff
@@ -0,0 +1,195 @@
+import jqueryConfig from "eslint-config-jquery";
+import importPlugin from "eslint-plugin-import";
+import globals from "globals";
+
+export default [
+	{
+
+		// Only global ignores will bypass the parser
+		// and avoid JS parsing errors
+		// See https://github.com/eslint/eslint/discussions/17412
+		ignores: [ "external", "null.json", "simple.json" ]
+	},
+
+	{
+		files: [ "eslint.config.js", "build/**" ],
+		languageOptions: {
+			ecmaVersion: "latest",
+			globals: {
+				...globals.node
+			}
+		},
+		rules: jqueryConfig.rules
+	},
+
+	{
+		files: [ "src/**" ],
+		plugins: {
+			import: importPlugin
+		},
+		languageOptions: {
+			ecmaVersion: 2015,
+
+			// The browser env is not enabled on purpose so that code takes
+			// all browser-only globals from window instead of assuming
+			// they're available as globals. This makes it possible to use
+			// jQuery with tools like jsdom which provide a custom window
+			// implementation.
+			globals: {
+				jQuery: false,
+				window: false
+			}
+		},
+		rules: {
+			...jqueryConfig.rules,
+			"import/extensions": [ "error", "always" ],
+			"import/no-cycle": "error",
+			indent: [
+				"error",
+				"tab",
+				{
+					outerIIFEBody: 0,
+
+					// This makes it so code within if statements checking
+					// for jQuery features is not indented.
+					ignoredNodes: [ "Program > IfStatement > *" ]
+				}
+			],
+			"one-var": [ "error", { var: "always" } ],
+			strict: [ "error", "function" ]
+		}
+	},
+
+	{
+		files: [ "src/wrapper.js" ],
+		languageOptions: {
+			sourceType: "script",
+			globals: {
+				jQuery: false,
+				define: false,
+				module: false
+			}
+		},
+		rules: {
+			"no-unused-vars": "off",
+			indent: [
+				"error",
+				"tab",
+				{
+
+					// This makes it so code within the wrapper is not indented.
+					ignoredNodes: [
+						"Program > ExpressionStatement > CallExpression > :last-child > *"
+					]
+				}
+			]
+		}
+	},
+
+	{
+		files: [ "test/unit/**" ],
+		languageOptions: {
+			ecmaVersion: 2021,
+			sourceType: "script",
+			globals: {
+				...globals.browser,
+				jQuery: false,
+				QUnit: false,
+				url: false,
+				expectWarning: false,
+				expectNoWarning: false,
+				compareVersions: false,
+				jQueryVersionSince: false,
+				startIframeTest: false,
+				TestManager: false
+			}
+		},
+		rules: {
+			...jqueryConfig.rules,
+			"no-unused-vars": [
+				"error",
+				{ args: "after-used", argsIgnorePattern: "^_" }
+			]
+		}
+	},
+
+	{
+		files: [ "test/data/**" ],
+		ignores: [ "test/data/jquery-2.2.3.js", "test/data/qunit-start.js" ],
```

Comment:
Can we make test jQuery updates easier by changing this to:
```suggestion
		ignores: [ "test/data/jquery-*.js", "test/data/qunit-start.js" ],
```
?